source/psa-build.rst

```diff
@@ -0,0 +1,59 @@
+.. _psa-build:
+
+=============================================
+Build the Percona Scheduler admin tool
+=============================================
+
+In the command line, run the following statement initialize all
+submodules and retrieve the latest code from the submodule:
+
+.. sourcecode:: bash
+
+   $> git submodule update --init
+   
+Run the following command to build the scheduler submodule. 
+
+.. sourcecode:: bash
+
+   $> build_scheduler.sh
+   
+Percona Scheduler Admin has two main files. The ``pxc-scheduler-handler`` binary
+and the ``percona_scheduler_admin`` script.
+
+The ``pxc-scheduler-handler`` does the
+following:
+
+* Monitors the cluster's health
+* Processes the cluster's state and evaluates the scenario
+* Performs actions, such as failover in case of an incident
+
+The ``percona_scheduler_admin`` script runs and provides responses based on the
+configuration file for the ``pxc-scheduler_handler`` binary.
+
+Running ``pxc-scheduler-handler``  without ``percona_scheduler_admin`` requires
+manual intervention.
+
+We do not recommend running multiple instances of the same binary. If you
+attempt this task when an instance of
+``pxc-scheduler-handler`` is running the binary runs, but the following occurs:
+
+* Provide information manually
```

Comment:
This is wrong. We can remove this line.

Reply:
Remove how much? Just the Provide information manually line or everything from line 37 to 40?

Reply:
Remove the only line that says `* Provide information manually`